gapis/gfxapi/vulkan/vulkan.api

```diff
@@ -9132,6 +9142,7 @@ enum RecordingState {
   @unused VkDescriptorPoolCreateFlags     Flags
   @unused u32                             MaxSets
   @unused map!(u32, VkDescriptorPoolSize) Sizes
+  @unused map!(VkDescriptorSet, ref!DescriptorSetObject) DescriptorSets
```

Comment:
If you just make this a map of u32->VkDescriptorSet you will avoid the circular dependency. We probably don't need the refs to the Objects here.